pyanaconda/geoloc.py

```diff
@@ -475,7 +475,7 @@ def name(self):
     def _refresh(self):
         try:
             reply = self._session.get(self.API_URL,
-                                      timeout=constants.NETWORK_CONNECTION_TIMEOUT,
+                                      timeout=constants.GEOLOC_CONNECTION_TIMEOUT,
```

Comment:
Could please change the timeout for all backends?

Reply:
Indeed, oops! :P Should be fixed now. :)

Reply:
The wrong timeout is also used in the `GoogleWiFiLocationProvider` class and the `Geocoder` class.

Reply:
Actually, let's just scrap all that! :D 

That WiFi location and geocoding code was basically a proof of concept that we never documented as supported by the boot options & the API URLs no longer work anyway. I wanted to drop it as part of moving the geolocation code to a task but might as well do it now. :)

See: https://github.com/rhinstaller/anaconda/pull/3030